testing/shc/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: MrBiTs <mrbits@mrbits.com.br>
+# Maintainer: MrBiTs <mrbits@mrbits.com.br>
+pkgname="shc"
+pkgver="4.0.3"
+pkgrel=0
+pkgdesc="A generic shell script compiler."
+url="https://neurobin.org/projects/softwares/unix/shc/"
+arch="all"
+license="GPL"
+depends=""
+makedepends=""
+install=""
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/mrbitsdcf/shc/archive/${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	./configure
+	make
+}
+
+check() {
+	# Replace with proper check command(s)
+	:
```

Comment:
Replace with proper check command(s)

Or remove this function and add options="!check" with a comment explaining why test is disabled